exercises/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,7 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(a, b) {
```

Comment:
It might be nice to hint that this is the solutions to signing up to Codewars, as well. Or at least guide people to think where they have seen this before.